src/alfasim_sdk/_internal/context.py

```diff
@@ -65,14 +67,18 @@ class PipelineInfo:
     The PipelineInfo provides information about the geometry of a pipeline.
 
     PipelineSegmentInfo provides the following attributes:
-        name: Name associated with this ``Pipeline`` on ALFAsim
+    :param name: Name associated with this ``Pipeline`` on ALFAsim
+    :type name: str
 
-        edge_name: Name of the edge that this ``Pipeline`` is associated with.
+    :param edge_name: Name of the edge that this ``Pipeline`` is associated with.
+    :type edge_name: str
```

Comment:
All the `:param` and `:type` fixing done here (well done btw) was done automatically?

Just to comment that in the future I think it is better to use proper type annotations rather than `:type` directives.